radicle-cob/src/history.rs

```diff
@@ -85,6 +87,16 @@ impl History {
             .unwrap_or_default()
     }
 
+    /// Get the current history timestamp.
+    /// This is the latest timestamp of any tip.
+    pub fn timestamp(&self) -> Clock {
```

Comment:
The dangers of `type` :smile: 
```suggestion
    pub fn timestamp(&self) -> Timestamp {
```

Reply:
Hahaha indeed.